### ISSUE TITLE
Ordering of {define} parameters causes error

### ISSUE THREAD

**Issue by gwyndaf** [ChordPro/chordpro#255]
[define_issue.cho.txt](https://github.com/ChordPro/chordpro/files/9701726/define_issue.cho.txt)
processed with `chordpro -X` gives error for `Fadd2`, but none for `Gadd2`:
```
"define_issue.cho.txt", line 2, Incorrect number of finger settings (0, should be 6)
	{define: Fadd2 base-fret 1 frets x 3 3 2 1 3 fingers 0 3 3 2 1 4 display F<sup>add2</sup>}
```
So it looks like the position of `display` is significant, and works when it follows the chord name but precedes (`frets` and) `fingers`.

Elsewhere, I have `{define-ukulele: A† copy A@4 display A<sup>†</sup>}`, which passes with no problem.

That seems to imply parameters (where used) should be in the order `name  copy  display  base-fret frets fingers`

I realise `copy` and `display` are currently undocumented, so this might not necessarily be a software issue, and simply something to include as and when those features are documented.

v5.988_049

**Comment by sciurius** [comment]
Ah! I recently fixed `frets` and `fingers` to prevent them from gobbling up all remaining arguments but there was still a typo in the fingers case. Fixed in 0.5988_050.

**Comment by gwyndaf** [comment]
Tested and now works nicely.
